domain-model/src/main/java/uk/gov/hmcts/cmc/domain/utils/ResponseUtils.java

```diff
@@ -54,4 +54,9 @@ public static boolean isResponsePartAdmitPayImmediately(Response response) {
             .filter(Predicate.isEqual(PaymentOption.IMMEDIATELY))
             .isPresent();
     }
+
+    public static boolean isAdmissionResponse(Response response) {
```

Comment:
minor: just for consistency should prob be like the other methods: `isResponseAdmission`